server/liveblog/themes/themes.py

```diff
@@ -81,6 +83,21 @@ def update_registered_theme_with_local_files(self):
                 created.append(theme)
         return (created, updated)
 
+    def on_delete(self, deleted_theme):
+        blogs_service = get_resource_service('blogs')
```

Comment:
First, you should check if it's not the default theme from the general settings. (Don't confuse "default-theme" which is a theme name, and "default theme" which refers to the theme select as default theme in the general settings)